src/main/java/com/savvato/tribeapp/repositories/VerbRepository.java

```diff
@@ -9,4 +10,7 @@
 @Repository
 public interface VerbRepository extends CrudRepository<Verb, Long> {
     Optional<Verb> findByWord(String word);
+
+    @Query(nativeQuery = true, value = "select word from verb where id = ?")
+    Optional<String> findVerbById(Long id);
```

Comment:
`findById(id)`

Reply:
Will fix according to what we decide in Adverb conversation above.